rep-2001.rst

```diff
@@ -35,81 +35,599 @@ End-user entry points
+
+
+ROS Base
+""""""""
+
+::
+
+  - ros_base:
+      extends:  [ros_core]
+      packages: [geometry2, kdl_parser, robot_state_publisher,
+                 rosbag2, urdf]
+
+
+Desktop
+"""""""
+
+::
+
+  - desktop:
+      extends:  [ros_base]
+      packages: [action_tutorials_cpp, action_tutorials_interfaces,
+                 action_tutorials_py, angles, composition,
+                 demo_nodes_cpp, demo_nodes_cpp_native,
+                 demo_nodes_py, depthimage_to_laserscan,
+                 dummy_map_server, dummy_robot_bringup,
+                 dummy_sensors,
+                 examples_rclcpp_minimal_action_client,
+                 examples_rclcpp_minimal_action_server,
+                 examples_rclcpp_minimal_client,
+                 examples_rclcpp_minimal_composition,
+                 examples_rclcpp_minimal_publisher,
+                 examples_rclcpp_minimal_service,
+                 examples_rclcpp_minimal_subscriber,
+                 examples_rclcpp_minimal_timer,
+                 examples_rclcpp_multithreaded_executor,
+                 examples_rclpy_executors,
+                 examples_rclpy_minimal_action_client,
+                 examples_rclpy_minimal_action_server,
+                 examples_rclpy_minimal_client,
+                 examples_rclpy_minimal_publisher,
+                 examples_rclpy_minimal_service,
+                 examples_rclpy_minimal_subscriber, image_tools,
+                 intra_process_demo, joy, lifecycle, logging_demo,
+                 pcl_conversions, pendulum_control, pendulum_msgs,
+                 quality_of_service_demo_cpp,
+                 quality_of_service_demo_py, rqt_common_plugins,
+                 rviz2, rviz_default_plugins, teleop_twist_joy,
+                 teleop_twist_keyboard, tlsf, tlsf_cpp,
+                 topic_monitor, turtlesim]
+
+
+Galactic Geochelone (May 2021 - November 2022)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ROS Core
+""""""""
+
+::
+
+  - ros_core:
+      packages: [ament_cmake, ament_cmake_auto, ament_cmake_gmock,
+                 ament_cmake_gtest, ament_cmake_pytest,
+                 ament_cmake_ros, ament_index_cpp,
+                 ament_index_python, ament_lint_auto,
+                 ament_lint_common, class_loader, common_interfaces,
+                 launch, launch_ros, launch_testing,
+                 launch_testing_ament_cmake, launch_testing_ros,
+                 launch_xml, launch_yaml, pluginlib, rcl_lifecycle,
+                 rclcpp, rclcpp_lifecycle, rclpy,
+                 ros2cli_common_extensions, ros2launch,
+                 ros_environment, rosidl_default_generators,
+                 rosidl_default_runtime, sros2, sros2_cmake]
+      And at least one of the following rmw_implementation:
+      - Fast-RTPS: [Fast-CDR, Fast-RTPS, rmw_fastrtps]
+      - Connext: [rmw_connext, rosidl_typesupport_connext]
+      - Opensplice: [rmw_opensplice, rosidl_typesupport_opensplice]
```

Comment:
For Galactic, Opensplice was dropped and CycloneDDS was added.

Reply:
Fixed in https://github.com/ros-infrastructure/rep/pull/327/commits/59d0c03654ae9b95b1470150747a97bcb44be209.